### ISSUE TITLE
False positive für implicit div sniff

### ISSUE THREAD

**Issue by oliverklee** [sds/haml-lint#40]
The code in https://gist.github.com/oliverklee/1fdfaa5b4d699cfd31d9 creates the following warning with haml-lint 0.7.0:

```
app/views/devise/unlocks/new.html.haml:14 [W] ImplicitDiv: `%div` can be written as `.submit` since `%div` is implicit
```

This is the corresponding line:

```
%div= f.submit 'Resend unlock instructions'
```

The warning is incorrect because the code after the div isn't a ".submit class", but a "f.submit tag".
